app/main.py

```diff
@@ -1,3 +1,59 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float | int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> int | float | object:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        else:
+            raise TypeError("Unsupported operand type")
+
+    def __iadd__(self, other: int | float) -> int | float | object:
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        else:
+            raise TypeError("Unsupported operand type")
+        return self
+
+    def __mul__(self, scalar: int | float) -> int | float | object:
+        if isinstance(scalar, (int, float)):
+            return Distance(self.km * scalar)
+        else:
+            raise TypeError("Unsupported operand type")
+
+    def __truediv__(self, divisor: int | float) -> int | float | object:
+        if isinstance(divisor, (int, float)):
+            result = self.km / divisor
+            return Distance(round(result, 2))
+        else:
+            raise TypeError("Unsupported operand type")
+
+    def __lt__(self, other: int | float) -> int | float | object | bool:
+        return self.km < other.km if isinstance(other, Distance)\
+            else self.km < other
```

Comment:
You can improve the readability of the code by using parentheses to break the line instead of a backslash. Fix in all places

Reply:
not fixe. Fix in all places